extensions/blocks/opentable/opentable.php

```diff
@@ -0,0 +1,155 @@
+<?php
+/**
+ * OpenTable Block.
+ *
+ * @since 8.x
+ *
+ * @package Jetpack
+ */
+
+if ( jetpack_opentable_block_is_available() ) {
+	jetpack_register_block(
+		'jetpack/opentable',
+		array( 'render_callback' => 'jetpack_opentable_block_load_assets' )
+	);
+} else {
+	Jetpack_Gutenberg::set_extension_unavailable(
+		'jetpack/opentable',
+		'missing_plan',
+		array(
+			'required_feature' => 'opentable',
+			'required_plan'    => ( defined( 'IS_WPCOM' ) && IS_WPCOM ) ? 'value_bundle' : 'jetpack_premium',
+		)
+	);
+}
+
+/**
+ * Is the OpenTable block available on a given site
+ *
+ * @return bool True if the block is available, false otherwise.
+ */
+function jetpack_opentable_block_is_available() {
+	// For WPCOM sites.
+	if ( defined( 'IS_WPCOM' ) && IS_WPCOM && function_exists( 'has_any_blog_stickers' ) ) {
+		$site_id = jetpack_get_blog_id();
+		return has_any_blog_stickers( array( 'premium-plan', 'business-plan', 'ecommerce-plan' ), $site_id );
+	}
+	// For all Jetpack sites.
+	return Jetpack::is_active() && Jetpack_Plan::supports( 'opentable' );
+}
+
+/**
+ * Get the current blog ID
+ *
+ * @return int The current blog ID
+ */
+function jetpack_opentable_block_get_blog_id() {
+	if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+		return get_current_blog_id();
+	}
+	return Jetpack_Options::get_option( 'id' );
+}
+
+/**
+ * OpenTable block registration/dependency declaration.
+ *
+ * @param array $attributes    Array containing the OpenTable block attributes.
+ *
+ * @return string
+ */
+function jetpack_opentable_block_load_assets( $attributes ) {
+	Jetpack_Gutenberg::load_assets_as_required( 'opentable' );
+
+	// The OpenTable script uses multiple `rid` paramters,
+	// so we can't use WordPress to output it, as WordPress attempts to validate it and removes them.
+	// phpcs:ignore WordPress.WP.EnqueuedResources.NonEnqueuedScript
+	return '<script type="text/javascript" src="' . esc_url( jetpack_opentable_build_embed_url( $attributes ) ) . '"></script>';
```

Comment:
Should we use `Jetpack_Gutenberg::block_classes()` here and wrap the block into a container using those classes? It would allow the classes that can be entered here to be used:

![image](https://user-images.githubusercontent.com/426388/71984067-69016080-3228-11ea-9654-22e2527f0bc5.png)

Reply:
Good idea. Done ✅